### PR TITLE
Changes related to issue #73

### PR DESCRIPTION
**Pull request by eervin123**

### CODE REVIEW

**Comment by eervin123** [comment]
Forgot to say thank you to @kite8 for the suggestion

File: setup.py [hunk 1]
```diff
@@ -4,9 +4,15 @@
 import re
 
 
+# def local_file(filename):
```
Comment (timkpaine) [inline setup.py:7]: just remove the `local_file` function altogether do the version code in a `with open` context

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (eervin123) [reply setup.py:7]: Sorry, I'm out of my depth. I just thought this was a simple fix that I could help with. I probably will end up creating more work than it's worth.

**Comment by timkpaine** [comment]
moving to #115

**Comment by eervin123** [comment]
Thanks, sorry I couldn't be more helpful. Now I see your fix and understand how basic it was.

**Pull request closed without merging**